docs/index.md

```diff
@@ -839,6 +839,7 @@ outcome | object | The outcome of the transaction (what effects it had).
 *outcome.* ledgerVersion | integer | The ledger version that the transaction was validated in.
 *outcome.* indexInLedger | integer | The ordering index of the transaction in the ledger.
 *outcome.* deliveredAmount | [amount](#amount) | *Optional* For payment transactions, it is impossible to reliably compute the actual delivered amount from the balanceChanges due to fixed precision. If the payment is not a partial payment and the transaction succeeded, the deliveredAmount should always be considered to be the amount specified in the transaction.
+*outcome.* deliveredAmount | string | *Optional* The delivered amount is reported as 'unavailable' for partial payment transactions before 2014
```

Comment:
Ok, I think it's better to just omit it then, it will be less likely to produce bugs in the client code